homeassistant/components/dunehd/media_player.py

```diff
@@ -19,11 +24,15 @@
     STATE_PAUSED,
     STATE_PLAYING,
 )
+from homeassistant.core import HomeAssistant
 import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType, StateType
 
 from .const import ATTR_MANUFACTURER, DEFAULT_NAME, DOMAIN
 
-CONF_SOURCES = "sources"
+CONF_SOURCES: Final = "sources"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Import `PLATFORM_SCHEMA` as `PARENT_PLATFORM_SCHEMA`.
```suggestion
PLATFORM_SCHEMA: Final = PARENT_PLATFORM_SCHEMA.extend(
```